examples/tfengine/team.hcl

```diff
@@ -32,7 +32,7 @@ data = {
   secret_locations    = ["us-central1"]
 
   labels = {
-    "env" = "prod"
+    env = "prod"
```

Comment:
add one example to demonstrate overriding a label in an individual resource?

Reply:
Added labels to all supported resources.